addons/packages/app-toolkit/0.1.0/README.md

```diff
@@ -179,3 +187,21 @@ This example illustrates creating a simple Spring Boot web app on the locally de
     ```shell
     curl http://tanzu-simple-web-app.default.127-0-0-1.sslip.io/
     ```
+  
+## Troubleshooting
```

Comment:
Could we add a troubleshooting tip for when a user isn't able to see curl their workload? 
The issue typically is when they haven't exposed the 80 and 443 ports while creating the TCE unmanaged cluster.